.golangci.yml

```diff
@@ -1,3 +1,16 @@
+linters-settings:
+  revive:
+    rules:
+      - name: line-length-limit
+        arguments: [120]
```

Comment:
Because of the logic with revive, by doing so you are adding revive, but not the default revive checker, only this one is enabled

You can have a look at one my project a d you will see the other revive checker I would expect to see once revive is enabled

https://github.com/ccoVeille/golangci-lint-config-examples/tree/main/03-safe

Reply:
Yes, that's right. I explicitly only enabled the line length check. I propose to fully enable revive in separate PRs.

Currently, the revive linter works the same whether the section in "linters-settings" is filled or missing.

Reply:
It makes sense,vyes